app/models/user.rb

```diff
@@ -1,11 +1,18 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] I would kindly suggest you add the `:confirmable` to the devise as it was requested in the project requirement that a user should be able to ask for confirmation of email. and the confirmable attribute handles that functionality, kindly add it.